vql/linux/info.go

```diff
@@ -0,0 +1,64 @@
+// +build linux
+
+package linux
+
+import (
+	"context"
+	"syscall"
+
+	"github.com/Velocidex/ordereddict"
+
+	"www.velocidex.com/golang/velociraptor/acls"
+	"www.velocidex.com/golang/vfilter"
+	vql_subsystem "www.velocidex.com/golang/velociraptor/vql"
+)
+
+type LinuxInfoPlugin struct {
+}
+
+// Needed values:
+// TotalPhysicalMemory
+// TotalFreeMemory
+func (self LinuxInfoPlugin) Info(scope vfilter.Scope, type_map *vfilter.TypeMap) *vfilter.PluginInfo {
```

Comment:
Do you think this makes more sense to be a VQL function? It only ever has one row.

Also maybe not name it similar to the info() plugin as it is specifically sysinfo() system call? so maybe name it sysinfo()

Reply:
I renamed it to linux_sysinfo() for clarity.